config/samples/agent-install.openshift.io_v1beta1_agentserviceconfig.yaml

```diff
@@ -2,18 +2,33 @@
 apiVersion: agent-install.openshift.io/v1beta1
 kind: AgentServiceConfig
 metadata:
- namespace: assisted-installer
  name: agent
 spec:
- databaseStorage:
-  accessModes:
-  - ReadWriteOnce
-  resources:
-   requests:
-    storage: 10Gi
- filesystemStorage:
-  accessModes:
-  - ReadWriteOnce
-  resources:
-   requests:
-    storage: 100Gi
+  databaseStorage:
+    accessModes:
+    - ReadWriteOnce
+    resources:
+      requests:
+        storage: 10Gi
+  filesystemStorage:
+    accessModes:
+    - ReadWriteOnce
+    resources:
+      requests:
+        storage: 100Gi
+  # Use this if you need to configure mirror registry
+  # mirrorRegistryRef:
+  #   name: mirrorRegistryConfigMapName
+  osImages:
+    - openshiftVersion: "4.6"
+      version: "46.82.202012051820-0"
+      url: "https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.6/4.6.8/rhcos-4.6.8-x86_64-live.x86_64.iso"
+      rootFSUrl: "https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.6/4.6.8/rhcos-live-rootfs.x86_64.img"
+    - openshiftVersion: "4.7"
+      version: "47.83.202103251640-0"
+      url: "https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.7/4.7.7/rhcos-4.7.7-x86_64-live.x86_64.iso"
+      rootFSUrl: "https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.7/4.7.7/rhcos-live-rootfs.x86_64.img"
+    - openshiftVersion: "4.8"
+      version: "47.83.202103251640-0"
+      url: "https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.7/4.7.7/rhcos-4.7.7-x86_64-live.x86_64.iso"
```

Comment:
I think this has been updated to use the links from https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/pre-release/4.8.0-fc.3/

Reply:
I think I need to make this be updated from the "default_ocp_versions". I'm going to update this manually for now and handle the scripting piece in a follow-up.